kuzzleio/kuzzle
# Add test for duplicates errors codes in config files

## What does this PR do ?

This PR adds a test for errors config files by testing if `domains` codes are unique.
Same for subdomains, and errors.

### How should this be manually tested?

Duplicate a code either for the `domain`, `subdomain` or an `error`, then start Kuzzle.
For instance you can open the `api.json` file, and give the same code for `document` and `index` subdomains
You should have an error
Modifying these json config files doesnt trigger the hot reload
### Other changes

Finds duplicate and fix it
